ATM_Selenium_Training/src/test/java/com/epam/atm/selenium/NewAppTest.java

```diff
@@ -0,0 +1,72 @@
+package com.epam.atm.selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.regex.Pattern;
+
+public class NewAppTest extends BaseTest {
+
+    @Test
```

Comment:
Test has also a description where u can add more info

Reply:
I know, thank you